dangerfile.js

```diff
@@ -131,5 +156,9 @@ const androidSnapshotsWithIosTokens = fileChanges.filter((filePath) => {
 
 if (androidSnapshotsWithIosTokens.length > 0) {
   // eslint-disable-next-line max-len
-  fail(`iOS tokens have been found in the following Android snapshots:\n  - ${androidSnapshotsWithIosTokens.join('\n  - ')}`);
+  fail(
+    `iOS tokens have been found in the following Android snapshots:\n  - ${androidSnapshotsWithIosTokens.join(
```

Comment:
^^^ I reckon some of our console formatting is going to be screwed up with this change.

Reply:
That case seems fine though